staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -1348,7 +1348,7 @@ type InterfaceBindingMethod struct {
 	DeprecatedMacvtap *DeprecatedInterfaceMacvtap `json:"macvtap,omitempty"`
 	// Deprecated, please refer to Kubevirt user guide for alternatives.
 	// +optional
-	Passt *InterfacePasst `json:"passt,omitempty"`
+	DeprecatedPasst *DeprecatedInterfacePasst `json:"passt,omitempty"`
```

Comment:
Please keep it consistent with macvtap.
I.e. change the documentation comet to:
```
// DeprecatedPasst is an alias to the deprecated Passt interface,
// please refer to Kubevirt user guide for alternatives.
// Deprecated: Removed in v1.3
// +optional
```

Reply:
Done